modules/genai-ecosystem/pages/llm-graph-builder-deployment.adoc

```diff
@@ -12,7 +12,8 @@ include::_graphacademy_llm.adoc[]
 == Prerequisites
 
 You will need to have a Neo4j Database V5.15 or later with https://neo4j.com/docs/apoc/current/installation/[APOC installed^] to use this Knowledge Graph Builder.
-You can use any https://neo4j.com/aura/[Neo4j Aura database^] (including the free tier database). Neo4j Aura automatically includes APOC and run on the latest Neo4j version, making it a great choice to get started quickly. You can also use the free trial in https://sandbox.neo4j.com[Neo4j Sandbox^], which also includes Graph Data Science.
+You can use any https://neo4j.com/aura/[Neo4j Aura database^] (including the free tier database). Neo4j Aura automatically includes APOC and run on the latest Neo4j version, making it a great choice to get started quickly. 
+You can also use the free trial in https://sandbox.neo4j.com[Neo4j Sandbox^], which also includes Graph Data Science.
 
 [CAUTION]
 ====
```

Comment:
Now able to run with Neo4j desktop, this warning should be change to:

If want to use https://neo4j.com/download/[Neo4j Desktop^] instead, when passing the Neo4j Connection URI in the frontend to connect to your local Neo4j database you will have to use host.docker.internal:7687 instead of localhost:7687 .